database/migrations/2013_11_15_190357_create_licenses_table.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+
+class CreateLicensesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
```

Comment:
Why is this here? This is a migration from 2013

Reply:
this one is a mistake from me. I mistakenly added the file because I didn't fast forward the repo after initial forking of the repo.

To be deleted